pages/docs/cloud.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: "Grouparoo Cloud Integration"
+date: "2021-11-08"
+pullQuote: "An easy-to-follow guide on integrating your Code Config repository with Grouparoo Cloud"
+---
+
+We're excited to share our Cloud offering! This guide should cover everything you need to get integrated
+using your own repository with Grouparoo Cloud.
+
+### Step 1 - Getting to your Project
+
+Sign in to your [Cloud](http://cloud.grouparoo.com/) account, or create a new one.
+
+#### If a new account
+
+Follow the steps shown on Cloud to create an Organzation and Project.
+
+#### If an existing account, but a new Grouparoo Project
+
+Use the Project dropdown to create a new Project.
+screenshot here
+
+#### If an existing account, and project
+
+You're set, follow the steps below!
+
+### Step 2 - Acquring an API Key
+
+Get your API Key. This can be found on your Project's primary page by clicking the Key icon near
+the top right of the page.
+API Keys are created with your project! Press the Copy button to copy the API Key.
+
+### Step 3 - Creating your Github action
+
+Note: Until Cloud is up and running, be sure to run the `grouparoo-infra-apps -> cloud-terraformer` Github action with your project's name!
+
```

Comment:
This is an internal detail that should not be in the public docs

Reply:
For sure. This is here so I don't forget during future testing and won't make it into the final version.